Sources/SwiftParser/Statements.swift

```diff
@@ -906,7 +916,7 @@ extension Parser {
         .poundIfKeyword, .poundErrorKeyword, .poundWarningKeyword,
         .poundEndifKeyword, .poundElseKeyword, .poundElseifKeyword
       ])
-        && !self.atStartOfStatement() && !self.atStartOfDeclaration() {
+        && !self.atStartOfStatement() && !self.atStartOfDeclaration() && !self.withLookahead({ $0.atStartOfSwitchCaseItem() }) {
```

Comment:
We should fix this in `atStartOfStatement`. This is ultimately a regression there because we dropped the clause that checks for `(@attr)* <identifier>`

Reply:
Do you think that case labels should return `true` for `atStartOfStatement`? Because in my opinion these are not statements. If anything, I’d say that case labels are declarations, but even that I don’t think they are.

Reply:
I would say yeah they're not real statements, but if we intend to match the behavior of the legacy parser, then we have to consider them a part of that predicate so the rest of the parser picks this up too.